### ISSUE TITLE
Cannot import sv_data after installation

### ISSUE THREAD

**Issue by shahmj** [PapenfussLab/sv_tools#1]
Hi,

I installed this tool and am able to import sv_diagram but get an import error when importing sv_data.  I'm wondering if this is a problem with my install or a bug in general?

Thanks for you help.
Minita

**Comment by shahmj** [comment]
Just figured out the issue, the script data.py should actually be sv_data.py and should be named correctly in the git push.

**Comment by Toseph** [comment]
Wanted to second Shahmj's comment. 

Came across the error when trying to do the following:
```
$ python
Python 2.7.10 (default, Nov  6 2015, 15:14:57)
[GCC 4.9.2 20150212 (Red Hat 4.9.2-6)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from sv_tools import sv_data
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name sv_data
```

To fix it, tried the following:
```

cd /path/to/sv_tools_git_pull
cd sv_tools
mv data.py sv_data.py
```

**Comment by l-d-s** [comment]
Thanks for pointing this out; fixed with [b2134b6](https://github.com/PapenfussLab/sv_tools/commit/b2134b6c2e0b820ea203e774d6f051f6a4ac81d2). Let me know of any other issues or improvements.
